go.mod

```diff
@@ -61,3 +61,6 @@ replace k8s.io/component-base => k8s.io/component-base v0.21.0
 replace k8s.io/component-helpers => k8s.io/component-helpers v0.21.0
 
 replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.21.0
+
+// WIP
+replace k8s.io/klog/v2 => github.com/pohly/klog/v2 v2.4.1-0.20210527141230-ac596814502c
```

Comment:
What's the plan for this?

Reply:
I proposed to have the code in klog: https://github.com/kubernetes/klog/pull/240

It's currently on hold because the logr API changes need to be dealt with first.